site/_scss/_global.scss

```diff
@@ -23,5 +23,9 @@ main:focus {
 }
 
 video {
+  // These custom properties are set by the Video shortcode if both the width
+  // and height attributes are set.
+  aspect-ratio: var(--vid-width) / var(--vid-height);
```

Comment:
What if these attributes are not set? Should be add fallback values or simply allow this value to become invalid?

Reply:
The fallback would be the default object size of 300x150 right?

Reply:
Falling back to invalid, which is a no-op, is probably more compatible with existing content on the site.